Kudu.Core/SiteExtensions/FeedExtensions.cs

```diff
@@ -52,9 +52,9 @@ public static class FeedExtensions
                 }
                 else
                 {
-                    // User provided simple string: treat it as a title. This is not ideal behavior, but
+                    // User provided simple string: treat it as a id. This is not ideal behavior, but
                     // is the best we can do based on how nuget.org works
-                    searchTerm = $"tags:AzureSiteExtension title:\"{searchTerm}\"";
+                    searchTerm = $"tags:AzureSiteExtension id:\"{searchTerm}\"";
```

Comment:
I'm switching this to ```id:```, since the functional tests pass a package Id instead of a title in places.  If this is too much of a breaking change, I can change the functional tests to pass package Titles instead.

Reply:
@EricSten-MSFT you can assume that no one is using the nuget.org branch of the current code, so don't worry about breaking changes

Reply:
@EricSten-MSFT as I'm trying to finally make this transition happen, this change from `title` to `id` seems problematic, because it cause search to only work for exac match. e.g. compare:

- Search by 'counter' successfully finds filecounter package when using `title:counter`: https://www.nuget.org/packages?q=tags%3AAzureSiteExtension+title%3Acounter
- The same doesn't find it when using `id:counter`: https://www.nuget.org/packages?q=tags%3AAzureSiteExtension+id%3Acounter

If the only issue here was a functional test, I think we should revert back to title. Do you know what test was failing?

Reply:
Ok, I moved back to title and updated the tests. 59d028358398e636611ad46e9323846abde1f535